docs/CONTRIBUTING.md

```diff
@@ -93,6 +93,10 @@ Unsure where to begin contributing? You can start by looking through these good
 * Do not include issue numbers in the first line of your commit message
 * Reference issues and pull requests liberally after the first line
 * [Verify your commits][signing-commits] before creating a pull request
+* Standardize your commits:
+    - use `yarn commit` to write the commit message with the interactive guide
+    - then check the message with `yarn commitmsg`
```

Comment:
Actually, `yarn commitmsg` always get executed after `yarn commit` because we use a package called [husky] that takes care of precommits and so on.

[husky]: https://github.com/typicode/husky

Reply:
this warning message keeps coming up:
```
husky > can't find npm in PATH, skipping precommit script in package.json
husky > can't find npm in PATH, skipping commitmsg script in package.json
```
sheikah use `yarn` for development, but how does the `npm` request cut in?

Reply:
What's your OS and git client? Most occurences of this error I can find are related to the GitHub Desktop client.

Reply:
os: Linux 4.14.47-manjaro, and I work with command line.

Reply:
Cool. Great to know you use Manjaro too. It's the official distro at our HQ :sunglasses: